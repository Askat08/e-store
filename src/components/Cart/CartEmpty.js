import React from "react";

export default function CartEmpty() {
  return (
    <div className="conainer m-5">
      <div className="row">
        <div className="col-10 mx-auto text-center">
          <h1>Cart is empty</h1>
        </div>
      </div>
    </div>
  );
}
