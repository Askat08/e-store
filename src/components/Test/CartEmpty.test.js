import React from "react";
import { shallow } from "enzyme";
import CartEmpty from "../Cart/CartEmpty";

describe("CartEmpty component test", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CartEmpty />);
  });

  it("shallow CartEmpty component", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("div has proper classname", () => {
    expect(wrapper.find("div").at(0).hasClass("conainer m-5")).toBeTruthy();
    expect(wrapper.find("div").at(1).hasClass("row")).toBeTruthy();
    expect(
      wrapper.find("div").at(2).hasClass("col-10 mx-auto text-center")
    ).toBeTruthy();
  });

  it("h1 has correct text", () => {
    expect(wrapper.find("h1").text()).toEqual("Cart is empty");
  });
});
