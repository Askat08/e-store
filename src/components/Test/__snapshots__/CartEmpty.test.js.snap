// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartEmpty component test shallow CartEmpty component 1`] = `
<div
  className="conainer m-5"
>
  <div
    className="row"
  >
    <div
      className="col-10 mx-auto text-center"
    >
      <h1>
        Cart is empty
      </h1>
    </div>
  </div>
</div>
`;
